widgetssdk/src/main/AndroidManifest.xml

```diff
@@ -87,7 +87,9 @@
             android:name=".core.screensharing.MediaProjectionService"
             android:foregroundServiceType="mediaProjection" />
 
-        <service android:name=".core.notification.NotificationRemovalService" />
+        <service
+            android:name=".core.notification.NotificationRemovalService"
+            android:exported="false" />
```

Comment:
I am not sure I understand how this effects the issue. If I am not mistaken the default value is 'false' anyways.
https://developer.android.com/guide/topics/manifest/service-element

Reply:
It doesn't affect the issue, I will remove it, thank you for pointing this out.